bubble_sort.rb

```diff
@@ -0,0 +1,55 @@
+def bubble_sort(array)
+  is_sorted = false
+
```

Comment:
This blank line and others within your code seem to be unnecessary.

Reply:
**Noted**